easybuild/easyconfigs/d/dammit/dammit-0.3.2-intel-2017a-Python-2.7.13.eb

```diff
@@ -0,0 +1,82 @@
+easyblock = 'Bundle'
+
+name = 'dammit'
+version = '0.3.2'
+versionsuffix = '-Python-%(pyver)s'
```

Comment:
Either include `Python`, `Perl` & `Ruby` in the `versionsuffix`, or none of them...

Since the `versionsuffix` would become very long here, I consider it OK to just drop it here as an exception to our rule of including `Python`/`Perl` version in there...

Reply:
@boegel I included `Python`, because dammit is actually written in `Python`. It have deps needs `Ruby` and `Perl`. Anyway, I will drop all of them.

Reply:
Well, OK, in that case I guess it makes sense to keep it.

Python is definitely way more common than Perl or Ruby, so it kind of makes sense to make only that one explicit in the `versionsuffix`...